docs/how-to/rocks/convert-to-pebble-layer.rst

```diff
@@ -29,7 +29,7 @@ A `Pebble layer
 <https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/
 layer-specification/>`_
 is composed of metadata, checks and services. The latter is present in
-``rockcraft.yaml`` as a `top-level field
+the project file as a `top-level key
 <https://canonical-rockcraft.readthedocs-hosted.com/en/latest/reference/
```

Comment:
note: unrelated but it looks like this link is broken.

Reply:
no idea why this isn't an internal reference, probably an oversight